scripts/download-blog-posts.data.json.mjs

```diff
@@ -2,12 +2,20 @@ import fs from "fs"
 import fetch from "node-fetch"
 
 const isMainnet = process.env.NEXT_PUBLIC_SCROLL_ENVIRONMENT === "Mainnet"
-const POSTS_URL = `https://blog.scroll.cat/api/posts/${isMainnet ? "published" : "preview"}/data.json`
+
+function buildPostURL(hostType) {
+  return `https://blog.scroll.cat/api/posts/${isMainnet ? "published" : "preview"}/${hostType}/data.json`
+}
 
 async function fetchPosts() {
-  await fetch(POSTS_URL, { headers: { Origin: "https://scroll.io" } })
-    .then(res => res.json())
-    .then(json => fs.writeFileSync("./src/app/blog/[blogId]/data.json", JSON.stringify(json, null, 2)))
+  await Promise.all([
+    fetch(buildPostURL("scroll.io"))
+      .then(res => res.json())
+      .then(json => fs.writeFileSync("./src/app/blog/[blogId]/data.json", JSON.stringify(json, null, 2))),
+    fetch(buildPostURL("research.scroll.io"))
+      .then(res => res.json())
+      .then(json => fs.writeFileSync("./src/research-blog.data.json", JSON.stringify(json, null, 2))),
```

Comment:
looks like the `.md` files in `src/assets/blog` are no longer needed. Can we delete these files and manage both `scroll.data.json` and `research.data.json` here instead?

Reply:
it's fixed, assets are now in `src/assets/*.blog.data.json`